### DISCUSSION TITLE
-clean images/folders

### DISCUSSION THREAD

**Discussion by custard2022** [AllskyTeam/allsky-modules#111] [General]
Hello,
Trying to figure out why the -clean folders exist and cannot be seen or removed using the WebUI.

In Allsky folder:
![Screenshot_20240306_142602](https://github.com/AllskyTeam/allsky/assets/40525844/3e311265-7d9c-4cf0-842a-9ebf30a90edc)
In WebUI: 
![Screenshot_20240306_142810](https://github.com/AllskyTeam/allsky/assets/40525844/227eec95-f9b8-4ef8-bb43-51ca216ca91b)

I had it set to Daytime Capture, but __NOT__ Daytime Save, the folders and images get saved, but cannot be removed or seen in the WebUI, taking up space unless manually deleted.

My question is:
1. What option does it make them save to file, and how to remove them?
2. What option is there to NOT save them, but only preview the camera Live without saving thousands of pictures?

**Comment by EricClaeys** [comment]
I've never saved an intermediate image so haven't see those directories before.  "intermediate" images are saved there via the "Saves an intermediate image" module in the Module Manager.  You can change the location of those images.
The rest of the WebUI doesn't know those directories exist, mainly because the Module Manager is fairly new and we're still improving it.

@Alex-developer, FYI - we should integrate these folders into the rest of Allsky, e.g., the WebUI should show them and DAYS_TO_KEEP should remove them.

![image](https://github.com/AllskyTeam/allsky/assets/83164203/79dd328f-1f7e-42a5-944b-aa3c313f99ad)

**Reply by custard2022** [reply]
Thank you, that was it!

**Reply by Alex-developer** [reply]
I have moved this to the modules repo and created an issue for it to be 'fixed'

**Reply by EricClaeys** [reply]
@Alex-developer, what does the "fix" entail?  If it's not trivial, let's wait until the release after v2024.xx.xx, ok?

**Reply by Alex-developer** [reply]
Eric, it will be after the next release as this whole process needs some thought

**Comment by Alex-developer** [comment]
Closing this as it will be looked at under the issue in a future release
